docs/resources/content_translation.md

```diff
@@ -0,0 +1,68 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "metabase_content_translation Resource - terraform-provider-metabase"
+subcategory: ""
+description: |-
+  A Metabase content translation dictionary. This resource manages the translation dictionary for Metabase Enterprise Edition. The dictionary content is stored as a hash in the state for efficiency.
```

Comment:
I'm not sure the hash brings any efficiency. Also, the entire content is stored in the state as well.

Reply:
You're right, it's a duplicate, I just removed the hash part.